README.md

```diff
@@ -35,6 +35,10 @@ to C++ by the [aws-crt-cpp](https://github.com/awslabs/aws-crt-cpp) package.
 
 #### Automatically Build and Install AWS Dependencies
 
+Note: `-DCMAKE_INSTALL_PREFIX` can be any path where the project needs to be installed. Taking sdk-cpp-workspace as an example here. And `--config` is only REQUIRED for multi-configuration build tools.
```

Comment:
"multi-configuration build tools"

I'd add something like "(VisualStudio/MsBuild being the most common)"

Also I'd keep this note only in the Windows section and remove the "--config" option entirely from the MacOS/Linux cmake invocation.

Reply:
Just to clarify should all of line 38 be kept in the windows section? or just second sentence about the "multi-configuration build tools"?

Reply:
Just the last fragment, sorry about that.  The install prefix note should stay general to all platforms.